### PR TITLE
neomutt: 20200626 -> 20200814

### PR DESCRIPTION
**Pull request by Ma27**
###### Motivation for this change

<del>:warning: **Caution:** `20200807` is only a development release and not considered stable, so please *don't* merge! I already had to apply a patch to fix a segfault on startup.</del>

<del>This is *only* intended to be used by power-users who want to try out the latest version. I don't consider this mergable until the 20.09 branchoff happened, after that we can discuss how to proceed with this.</del>

ChangeLog: https://github.com/neomutt/neomutt/releases/tag/20200807, https://github.com/neomutt/neomutt/releases/tag/20200814

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Ma27** [comment]
@GrahamcOfBorg eval

**Comment by Ma27** [comment]
Today's release is marked as stable again and thus we could merge this PR now. Already using the new neomutt version on my machine :)

**Comment by Ma27** [comment]
cc @flokli who also might want to take a look :)

File: pkgs/applications/networking/mailreaders/neomutt/default.nix [hunk 1]
```diff
@@ -1,18 +1,18 @@
 { stdenv, fetchFromGitHub, gettext, makeWrapper, tcl, which, writeScript
 , ncurses, perl , cyrus_sasl, gss, gpgme, kerberos, libidn, libxml2, notmuch, openssl
 , lmdb, libxslt, docbook_xsl, docbook_xml_dtd_42, elinks, mailcap, runtimeShell, sqlite, zlib
-, glibcLocales
+, glibcLocales, fetchpatch
```
Comment (flokli) [inline pkgs/applications/networking/mailreaders/neomutt/default.nix:4]: The `fetchpatch` isn't needed anymore.

**Review by flokli — CHANGES_REQUESTED** [review]
From a quick glance, LGTM otherwise.

File: pkgs/applications/networking/mailreaders/neomutt/default.nix (hunk 1 above)
Comment (Ma27) [reply pkgs/applications/networking/mailreaders/neomutt/default.nix]: Fixed :-)

**Comment by alyssais** [comment]
@GrahamcOfBorg build neomutt

**Pull request merged**